Jasmine/LevelDesignerViewController.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+
+class LevelDesignerViewController: UIViewController {
+
+    @IBOutlet private var levelNameField: UITextField!
+
+    /// The order of the game mode on the gameModeControl
+    private static let gameModeOrder: [GameMode] = [.tetris, .sliding, .swapping]
+    @IBOutlet private var gameModeControl: UISegmentedControl!
+
+    /// The order of the game type on the gameTypeControl
+    private static let gameTypeOrder: [GameType] = [.chengYu, .ciHui]
+    @IBOutlet private var gameTypeControl: UISegmentedControl!
+
+    /// The formatted text for the select phrases button, with %d representing the phrases count
+    private static let selectPhrasesButtonText = "Select phrases (%d currently)"
+    @IBOutlet private var selectPhrasesButton: UIButton!
+
+    /// The levels data to add to or update. To be set by LevelSelectionViewController
+    var levels: Levels?
```

Comment:
Can the VC receive the "Levels" and "Level" ADT? note that the level selector VC only has GameInfo object that can be used to segue into this view.